static/includes/apps/AppInstallWizardACLConfiguration.md

```diff
@@ -1,20 +1,27 @@
 &NewLine;
 
-Select **Enable ACL** to show the ACL and ACE Entries options.
+Select **Enable ACL** to show the ACL and **ACE Entries** options.
 
-Enter or browse to select the dataset path. 
+{{< expand "Configuring ACE Entries" "v" >}}
+Enter or browse to select the dataset and populate **Host Path**.
+
+{{< trueimage src="/images/SCALE/Apps/InstallAppStorageConfigurationACLandACESettings.png" alt="Host Path ACL and ACE Settings" id="Host Path ACL and ACE Settings" >}}
+
+Next, click **Add** to the right of **ACL Entries** to show the permissions settings.
+Set **ID Type** to **Entry is for a USER** or **Entry is for a GROUP**.
+If you configured a group in TrueNAS that you want to give access to instead of a single user, set the ID to the group option and enter the GID for that group.
 
-Next click **Add** to the right of **ACL Entries** to show the permissions settings.
-Set **ID Type** to **Entry is for a USER**.
 Enter the UID as one of the following:
-* The default apps user which is either **473** for apps in the stable train or **568** for apps in the **enterprise** or **community** train.
-* The new user ID for the TrueNAS user created to serve as the app administrator.
+* The default apps user, which is **473** for apps in the stable train, **568** for apps in the **enterprise** or **community** trains, **999** for postgres storage volumes, or **0** if running as root.
```

Comment:
The `473` user is only used on minio. But its not the rule for stable apps.

The rule is something like that:

All apps that can be run as any non-root user, default to `568`.

Apps that can run only as a specific non-root user, are run as this specific user. Which there is no option in the UI to change that.

Apps that need root privileges, run as root. Which there is no option in the UI to change that.

Postgres and MariaDB on all apps runs as 999:999, regardless of the Run As setting (This only affects the main container).

Reply:
I'll get these updates made! Thanks for the feedback!